src/mailmanclient/_client.py

```diff
@@ -255,6 +256,33 @@ def delete_list(self, fqdn_listname):
         response, content = self._connection.call(
             'lists/{0}'.format(fqdn_listname), None, 'DELETE')
 
+    def count_unsubscriber(self, list_identifier, start_date=None, stop_date=None):
```

Comment:
Maybe some comments about what this function that and specially about the parameters would be helpful. I take it start_date and stop_date are the "from" and "to" that the admin would choose to get the numbers of unsubscribes during a period of time, but it's always good to document this.